esp/esdllib/esdl_script.cpp

```diff
@@ -2797,6 +2974,54 @@ class CEsdlFunctionRegister : public CInterfaceOf<IEsdlFunctionRegister>
     }
 };
 
+class CEsdlOperationMessenger : public IEsdlOperationMessenger
```

Comment:
Can we name this CEsdlOperationDefaultTraceMessenger?  If we ever use it for anything else we can rename it then.